### PR TITLE
chore(groups): Get the image sizes from the icon_sizes config

### PR DESCRIPTION
**Pull request by Srokap**
Replaces https://github.com/Elgg/Elgg/pull/7265 as squashed version of the same changes.

### CODE REVIEW

File: mod/groups/actions/groups/delete.php [hunk 1]
```diff
@@ -20,11 +20,11 @@
...
 	$owner_guid = $entity->owner_guid;
 	$prefix = "groups/" . $entity->guid;
-	$imagenames = array('.jpg', 'tiny.jpg', 'small.jpg', 'medium.jpg', 'large.jpg');
+	$imagenames = elgg_get_config('icon_sizes');
 	$img = new ElggFile();
 	$img->owner_guid = $owner_guid;
-	foreach ($imagenames as $name) {
-		$img->setFilename($prefix . $name);
+	foreach ($imagenames as $name => $value) {
+		$img->setFilename($prefix . $name.'.jpg');
```
Comment (jdalsem) [inline mod/groups/actions/groups/delete.php]: needs an extra space around the concat... should scrutinizer not catch this?

**Comment by juho-jaakkola** [comment]
What happens when missing master size is attempted to be displayed? Does Elgg assume they exist because ```icontime``` has a value?

**Comment by jdalsem** [comment]
it will revert to medium as there is no file available https://github.com/Elgg/Elgg/blob/1.10/mod/groups/icon.php#L43

**Comment by juho-jaakkola** [comment]
Ok, LGTM

**Comment by mrclay** [comment]
Merged in 75c62f204b91c2761c6810539d1b9348e10462cc

**Pull request closed without merging**